### ISSUE TITLE
Incompatibility issue with other fields introduced on version `^1.7.0` 

### ISSUE THREAD

**Issue by milewski** [optimistdigital/nova-translatable#47]
Hi, something might have changed between version `1.6.12` and `1.7.0` that causes the package https://github.com/froala/nova-froala-field to not work anymore within a flexible-field-content field.. 

If use version `1.6.12` = Everything works fine
If I use version `1.7.0` = The content of the field is completely incorrect

Code example:

```php
Flexible::make('Test', 'Test')->addLayout('Test', 'Test', [

    Froala::make('Content', 'content')->alwaysShow()->translatable(),

])
````

**Comment by KasparRosin** [comment]
Hey! Just released version 1.7.2 that should fix this issue. 
Let me know if there are any other problems. I recently reworked `fill` method a bit to reduce duplicate code, so hopefully everything now works as it did before. 
Thank you for reporting this.

**Comment by milewski** [comment]
Hi, thanks! Just tested it is working now!
